tests/runfiles/linux.run

```diff
@@ -179,7 +178,8 @@ tests = ['cache_001_pos', 'cache_002_neg', 'canmount_001_pos',
     'share_mount_001_neg', 'snapdir_001_pos', 'onoffs_001_pos',
     'user_property_001_pos', 'user_property_003_neg', 'readonly_001_pos',
     'user_property_004_pos', 'version_001_neg', 'zfs_set_001_neg',
-    'zfs_set_002_neg', 'zfs_set_003_neg', 'property_alias_001_pos']
+    'zfs_set_002_neg', 'zfs_set_003_neg', 'property_alias_001_pos',
+	'mountpoint_003_pos']
```

Comment:
It looks like you've got some extra whitespace here and a few other places in the patch.